msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,203 @@
+
+Admins can create a detailed preapproval policy, based on the app ID, permissions, and the sensitivity of the accessed data. Preapproval of RSC permissions are designed for admins seeking to create advanced custom policies for their organization. 
+
+By default, the preapproval policies are managed by Microsoft, and we recommend that organizations maintain `ManagedByMicrosoft` state. This state allows Microsoft's security team to deliver the best security for your organization.
+
+> [!NOTE]
+> Preapproval of RSC permissions is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> Preapproval of RSC permissions and its operating procedures are subject to change.
+
+## Set up PowerShell to manage preapproval of RSC permissions
+
+Preapproval of RSC permissions is managed through Microsoft Graph PowerShell. You can learn more about managing Microsoft Teams with PowerShell [here](/powershell/microsoftgraph/get-started).
+
+Before you can create and manage preapproval policies, use `Connect-MgGraph` cmdlet to connect PowerShell to your tenant using Microsoft Graph. To create, manage, and delete RSC preapproval policies, add the following permissions: You will need Global admin level privleges to connect Graph with your organization for the first time.
+
+* `TeamworkAppSettings.ReadWrite.All`
+* `Policy.ReadWrite.Authorization`
+* `AppCatalog.Read.All`
+* `Policy.ReadWrite.PermissionsGrant`
+* `InformationProtectionPolicy.Read`
+* `Application.ReadWrite.All`
+
+The following is an example of the PowerShell set up to manage preapproval policies of RSC permissions:
+
+```powershell
+Connect-MgGraph -Scopes @('TeamworkAppSettings.ReadWrite.All', 'Policy.ReadWrite.Authorization', 'AppCatalog.Read.All', 'Policy.ReadWrite.PermissionGrant', 'InformationProtectionPolicy.Read', 'Application.ReadWrite.All')
+```
+
+## Allow RSC permissions for apps in your organization
+
+For both team and chat specific RSC permissions, admins can set four different states for their organization. Apart from `DisabledForAllApps` state, all the other states allow RSC permissions. These states can be set through `Set-MgBetaChatRscConfiguration` and `Set-MgBetaTeamRscConfiguration` cmdlets.
+
+For example, to allow RSC permissions for all unblocked apps in your organization, use the `Set-MgBetaTeamRscConfiguration -State EnabledForPreApprovedAppsOnly` cmdlets.
+
+The following are different states that allow and disallow RSC permission in your organization:
+
+|Configuration| Description|
+|---|---|
+|`ManagedByMicrosoft`|A dynamic policy administered by Microsoft. It might be updated based on best security practices. By default, this state enables RSC for all unblocked apps in the organization. |
+|`ApprovedForAllApps`|Users can consent to RSC permissions for any unblocked app in the organization. |
```

Comment:
States need to be fixed

Reply:
EnabledForAllApps, EnabledForPreApprovedAppsOnly, ManagedByMicrosoft, DisabledForAllApps